runtime/vm/visible.c

```diff
@@ -360,7 +379,10 @@ loadAndVerifyNestHost(J9VMThread *vmThread, J9Class *clazz, UDATA options)
 
 		/* If a problem occurred in nest host verification then the nest host value is invalid */
 		if (J9_VISIBILITY_ALLOWED == result) {
-			clazz->nestHost = nestHost;
+			*nestHostFound = nestHost;
+			if (cacheNestHostInClass) {
+				clazz->nestHost = *nestHostFound;
```

Comment:
In an earlier draft, this used `nestHost`, please keep it that way.
Same comment on line 398 with respect to `curClazz`.